website/docs/r/kubernetes_auth_backend_role.html.md

```diff
@@ -44,11 +44,16 @@ The following arguments are supported:
 
 * `audience` - (Optional) Audience claim to verify in the JWT.
 
+~> Please see [alias_name_source](https://www.vaultproject.io/api-docs/auth/kubernetes#alias_name_source)
+before setting this to something other its default value. There are **important** security
+implications to be aware of.
+ 
+* `alias_name_source` - (Optional, default: `serviceaccount_uid`) Configures how identity aliases are generated.
+   Valid choices are: `serviceaccount_uid`, `serviceaccount_name`. (vault-1.9+)
+
 ### Common Token Arguments
 
 These arguments are common across several Authentication Token resources since Vault 1.2.
-
-* `token_ttl` - (Optional) The incremental lifetime for generated tokens in number of seconds.
```

Comment:
Has this since been deprecated?

Reply:
I think `token_ttl` replaced `ttl` which has now been removed.

Reply:
We could probably go through and remove all of the chatter about `Vault 1.2...`

Reply:
Cool no problem, just wanted to confirm!